numpy/linalg/lapack_lite/python_xerbla.c

```diff
@@ -2,9 +2,6 @@
 #include "numpy/npy_common.h"
 #include "npy_cblas.h"
 
-#undef c_abs
-#include "f2c.h"
```

Comment:
Not included simply because it's not needed?

Reply:
Yes. The symbol rename also conflict here with BLAS_FUNC. This file is independent of lapack_lite, as it is also used when using a different blas.